### PR TITLE
popup-branch 

### PR DESCRIPTION
**Pull request by noelfoka**
In this branch, I did create:

- [ ] popup-branch
- [ ] popup.js file and edit it
- [ ] change my title html

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Noel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, however, you're the only person making commits to the popup-branch, Because this is a pair programming assignment there should be commits from your coding partner as well

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -519,5 +703,6 @@ <h2 class="form-header">Contact me</h2>
         </section>
         <!-- Script -->
         <script src="./main.js"></script>
+        <script src="./popup.js"></script>
```
Comment (joel-kalema) [inline index.html]: Good job working on the details popup window 👍, however, you still need to fix some parts 

- [ ] Kindly make sure you stuck to the design as much as possible (e.g., `font`, `colors`, `images`, `text`, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃. Also make sure the popup for the mobile version is displaying well 👍 

## Figma 

> | | [For desck](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1143&scaling=min-zoom&page-id=0%3A1) (click there) | | [For mobile](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-418&scaling=min-zoom&page-id=0%3A1) (click there) | |
> |-|---------|-|--------|-|
> | |Capture| |Capture| | 

## Your design

> | |For desck| |For mobile| |
> |-|---------|-|--------|-|
> | |Capture| |Capture| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi Team
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (joel-kalema) [inline index.html]: - [ ] Good job working on the details popup window 👍, however, make sure you stuck to the design as much as possible in the mobile version (`font`, `colors`, `images`, `text`, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.). Kindly, fix it 😃.  

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Capture| |Capture| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi Team
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @noelfoka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented animations 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**